content/javascript/concepts/promise/promise.md

```diff
@@ -81,3 +81,40 @@ promise
 
 // Output: Success: The promise was fulfilled!
 ```
+
+#### Async Await
+
+Starting from ES2017, JavaScript introduced the keywords "async" and "await", which is used to write promises in a cleaner way. This can be used as an alternative to other verbose solutions, such as promise chaining.
+
+Note that "await" is only usable in an asynchronous function, otherwise it will throw a SyntaxError.
+
+```js
```

Comment:
The code as written does not produce any output because the async function is never called. Fix this code snippet so it does produce an output.

Also, instead of including the output as a comment in the code, write it in a separate shell block after the code. Like this:
````
This example results in the following output:

```shell
Output from the code.
```
````